.gitignore

```diff
@@ -1,5 +1,6 @@
 # Compiled class file
 *.class
+test.md
```

Comment:
- [OPTIONAL] I can not still see the  `node_modules` directory inside your gitignore file as required by the first review.